src/math.ts

```diff
@@ -348,7 +347,7 @@ export const getPointsInBezierCurve = (
     t -= 0.05;
   }
   if (pointsOnCurve.length) {
-    if (!LinearElementEditor.isEqual(pointsOnCurve.at(-1)!, endPoint)) {
+    if (arePointsEqual(pointsOnCurve[1], endPoint)) {
```

Comment:
why this change?

Reply:
Reverted. I might remove this as well later if the midpoints calculated fine without the last point but for now I will keep it.